test/e2e/node_exporter_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 The Cluster Monitoring Operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestNodeExporterCollectorEnablement(t *testing.T) {
+
+	cm, err := f.KubeClient.CoreV1().ConfigMaps(f.Ns).Get(ctx, clusterMonitorConfigMapName, metav1.GetOptions{})
+
+	if err == nil {
+		cmBackup := cm.DeepCopy()
+		cmBackup.ObjectMeta.ResourceVersion = ""
+		cmBackup.ObjectMeta.UID = ""
+		cmBackup.ObjectMeta.CreationTimestamp = metav1.Time{}
+
+		t.Cleanup(func() {
+			f.MustCreateOrUpdateConfigMap(t, cmBackup)
+		})
+	} else if errors.IsNotFound(err) {
+		t.Cleanup(func() {
+			f.MustDeleteConfigMap(t, &corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      clusterMonitorConfigMapName,
+					Namespace: f.Ns,
+				},
+			})
+		})
+	} else {
+		t.Fatalf("failed to get configmap %s in namespace %s - %s", clusterMonitorConfigMapName, f.Ns, err.Error())
+	}
```

Comment:
do we need this pre-check?

Reply:
When CI executes this test, the config map does not exist. 
This part is to restore CMO config to the same state as before the test, in case other tests require the previous CMO config.